Doc/library/dis.rst

```diff
@@ -1352,6 +1352,13 @@ iterations of the loop.
       This opcode is now only used in situations where the local variable is
       guaranteed to be initialized. It cannot raise :exc:`UnboundLocalError`.
 
+.. opcode:: LOAD_FAST_LOAD_FAST (var_nums)
+
+   Pushes references to ``co_varnames[var_nums >> 4]`` and
+   ``co_varnames[var_nums & 15]`` onto the stack.
```

Comment:
I am bothered by this explanation since as noted in #122794 LOAD_FAST can access names outside of co_varnames and I assume the same is true here. If this fix is accepted, the same solution that will resolve #122794 will have to be applied here too.

Reply:
Totally agreed. Let's see what the outcome of https://github.com/python/cpython/issues/122794 is and apply it here.

Reply:
I don't think this PR necessarily needs to be help up by this. This can be fixed as part of https://github.com/python/cpython/issues/122794.